CHANGES.md

```diff
@@ -8,6 +8,16 @@
 
 [702]: https://github.com/boot-clj/boot/pull/702
 
+#### Breaking
+
+Updated nREPL to 0.4.3. The new nREPL has a new artefact coordinates
```

Comment:
`s/artefact/artifact` ?

Reply:
Not in my part of the world. 😆

Reply:
Sure, just noted it to keep it in line with maven spelling:
<img width="296" alt="screen shot 2018-07-26 at 4 21 09 pm" src="https://user-images.githubusercontent.com/2217181/43268078-f614ccb8-90ef-11e8-92b6-8ec87ff84d2e.png">

Also: https://github.com/boot-clj/boot/search?q=artifact&unscoped_q=artifact
vs.
https://github.com/boot-clj/boot/search?q=artefact&unscoped_q=artefact

Reply:
Yeah, yeah. I know everyone in programming uses American English spellings, but it's hard for me to remember this all the time. Especially when my spell checker highlights `artifact` as incorrect. :-)